packages/shared/lib/utils/error.ts

```diff
@@ -559,6 +559,13 @@ export class NangoError extends Error {
                 this.message = `Missing an account name for account login/signup.`;
                 break;
 
+            case 'resource_capped':
+                this.status = 400;
+                // TODO docs link
+                this.message =
+                    'Free accounts cannot have > 3 connections for integrations with active scripts. Upgrade or deactivate the scripts to create more connections (https://docs.nango.dev/relevant-link).';
```

Comment:
Not a good idea to document moving part in the error message (people rely on error message to automate stuff and we often forget to update them).
Something like `You have reached maximum number of connections. Check our documentation to know more (https://docs.nango.dev/relevant-link).` create far less problem in the long-run

Reply:
Updated to be less specific